src/components/contact/Contact.jsx

```diff
@@ -12,16 +12,16 @@ const Contact = ({ t }) => {
 
   return (
     <section className={styles.contact}>
-      <div className={styles.left_section}>
+      <section className={styles.left_section}>
```

Comment:
cool

Reply:
Is in this file everything ok (because I think it is marked as "request changes")?